blocks/header/header.css

```diff
@@ -18,249 +28,278 @@ header nav {
   height: var(--nav-height);
   padding-inline: var(--content-padding-inline);
   font-family: var(--body-font-family);
-}
 
-header nav[aria-expanded='true'] {
-  grid-template:
-    'hamburger brand' var(--nav-height)
-    'sections sections' 1fr
-    'tools tools' var(--nav-height) / auto 1fr;
-  overflow-y: auto;
-  min-height: 100dvh;
-}
+  &[aria-expanded='true'] {
+    grid-template:
+      'hamburger brand' var(--nav-height)
+      'sections sections' 1fr
+      'tools tools' var(--nav-height) / auto 1fr;
+    overflow-y: auto;
+    min-height: 100dvh;
+
+    @media (min-width: 1280px) {
+      min-height: 0;
+      overflow: visible;
+    }
+  }
 
-@media (min-width: 1280px) {
-  header nav {
+  @media (min-width: 1280px) {
```

Comment:
put media queries before other modifier, this piece of code should go to line 31

Reply:
Oh buy, how could've I missed that? 😢